### PR TITLE
Repair Cleanup

### PR DESCRIPTION
**Pull request by jonpas**

### CODE REVIEW

File: addons/repair/functions/fnc_isEngineer.sqf [hunk 1]
```diff
@@ -1,13 +1,16 @@
 /*
  * Author: Glowbal, KoffeinFlummi, commy2
- * Check if a unit is any engineer class
+ * Check if a unit is any engineer class.
  *
  * Arguments:
- * 0: The Unit <OBJECT>
- * 1: Class <NUMBER> <OPTIONAL>
+ * 0: Unit <OBJECT>
+ * 1: Class <NUMBER> (Optional)
```
Comment (jokoho48) [inline addons/repair/functions/fnc_isEngineer.sqf]: better (default: "Value")?

Comment (jonpas) [inline addons/repair/functions/fnc_isEngineer.sqf]: Meh, doesn't really matter.

Comment (jonpas) [inline addons/repair/functions/fnc_isEngineer.sqf]: Did it anyways. :banana:

Comment (MikeMatrix) [inline addons/repair/functions/fnc_isEngineer.sqf]: Using `default` will imply that is optional, but also give you a reference to what the default value is.
Using `optional` only gives you information that it is not required, but not about what happens when you don't specify it.

Comment (jokoho48) [inline addons/repair/functions/fnc_isEngineer.sqf]: and we can later use a tool for auto Wiki Creation :D

Comment (jonpas) [inline addons/repair/functions/fnc_isEngineer.sqf]: Yep yep I get it, here have a :banana: each!

**Pull request merged**